app/views/deceased_helpline.scala.html

```diff
@@ -14,70 +14,43 @@
  * limitations under the License.
  *@
 
-@()(implicit request: Request[_], user: controllers.auth.AuthedUser, messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever)
+@()(implicit request: Request[_], messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever)
 
 @import uk.gov.hmrc.urls.Link
 
 @main(
-    title = Messages("tai.deceased.title"),
+    title = Messages("tai.deceased.heading.title"),
     articleClasses = Some("full-width"),
-    authedUser = Some(user),
     gaCustomTitle = None
 ){
 
     <header class="page-header">
-        <h1 id="deceased-header" class="flush--bottom">@Messages("tai.deceased.heading", user.getDisplayName)</h1>
+        <h1 id="deceased-header" class="flush--bottom">@Messages("tai.deceased.heading.title")</h1>
     </header>
 
-    <p id="deceased-p1">@Messages("tai.deceased.information.p1", user.getDisplayName)</p>
+    <div class="grid-row">
+        <div class="grid grid-2-3">
```

Comment:
Just a note, looked into main_template which is using article. I believe it's possible to inject these classes; grid grid-2-3 classes to save overruling with using divs